src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private int range = 40;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private final String[] figures = {"square",
+            "rectangle",
+            "righttriangle",
+            "circle",
+            "isoscelestrapezoid"};
```

Comment:
The array of figure types should be a constant as well since it does not change. Consider declaring it as a public static final variable with an appropriate name in uppercase.